Casks/circleci-runner.rb

```diff
@@ -37,7 +37,7 @@
       end
     end
 
-    if not File.exists?(plistFile)
+    if not File.exist?(plistFile)
```

Comment:
We look to pin the xcode version, so I'm not sure why this changed, but with Ruby 3.2+, this method has been renamed.